### PR TITLE
[bitnami/spark] Added hostNetwork in the spark charts

### PR DESCRIPTION
**Pull request by a-nldisr**
Signed-off-by: Rogier Dikkes <1867776+a-nldisr@users.noreply.github.com>

**Description of the change**

The change adds hostNetwork to the spark worker and master. Gives people the ability to enable hostNetwork if they require host networking to connect to their workloads. 

**Benefits**

Enables hostNetwork in the bitnami spark chart. 

**Possible drawbacks**

The option to not use hostNetwork stays intact and therefore should have no impact. 

**Applicable issues**

None (yet), I can create an issue if requested. 

**Additional information**

The reason why I want to add this feature is that we are currently migrating from DCOS to Kubernetes. We use Spark to connect to various internal datastores which are reachable inside the DCOS cluster. In DCOS there is an ipvs based load balancer that offers internal URI's to connect from spark to the databases. While we are migrating our Spark workloads to Kubernetes we need to keep using the datastores. We therefore made an ipvs load balancer setup that bridges the Kubernetes cluster via IPVS to DCOS we however need to enable hostNetwork in all the pods that need to connect to the DCOS cluster. This feature was missing in the bitnami spark charts. 

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/spark/Chart.yaml [hunk 1]
```diff
@@ -22,4 +22,4 @@ name: spark
 sources:
   - https://github.com/bitnami/bitnami-docker-spark
   - https://spark.apache.org/
-version: 5.2.2
+version: 5.2.4
```
Comment (miguelaeh) [inline bitnami/spark/Chart.yaml:25]:
```suggestion
version: 5.3.0
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!

File: bitnami/spark/Chart.yaml (hunk 1 above)
Comment (a-nldisr) [reply bitnami/spark/Chart.yaml]: Updated the version as requested.

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**